byoeb-v1/byoeb/byoeb/scripts/auth.py

```
@@ -0,0 +1,228 @@
+                raise SystemExit(f"Invalid tenant-id (not a UUID): {args.tenant_id}") from exc
+        
+        integrations = list(integration_collection.find(query))
+        if not integrations:
+            print("No integrations found.")
+            return 0
+        
+        print(f"{'Platform':<15} {'Identifier':<20} {'Tenant ID':<40} {'Verify Token'}")
+        print("-" * 100)
+        for i in integrations:
+            verify_token = i.get('credentials', {}).get('verification_token', 'N/A')
+            print(f"{i.get('platform', 'N/A'):<15} {i.get('identifier', 'N/A'):<20} {str(i.get('tenant_id', 'N/A')):<40} {verify_token}")
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)